docs/release-process.md

```diff
@@ -56,28 +90,32 @@ Hotfixes are started off of master. Bugfixes off of stable.
   A new branch called release/x.x.x will be made for you and pushed to the repository.
 
 1. Operator images should be built and pushed for you with the build. Additionally there will be a github check that will create assets for you. You will need to publish the images in partner connect.
+  1. Verify the checks for the PR are passing.
+  1. Use the "stable-" assets from Generate Bundle workflow check to update partner connect. The bundle is a zip file with a zip inside of it. You will want to unzip the top file and submit the zip inside of it. Name will start with rhm-op-bundle.
+  1. You can use the `test-catalog-source` to install to your local CRC or to another openshfit to install the latest operator bundles for verification. You can just `kube apply -f` the file.
 
-1. Upload your generated bundle to [partner connect](#updating-partner-connect). And publish it when it passes.
+1. Upload your generated bundle to [partner connect](#updating-partner-connect). And publish it when it passes. More details are provided in the [updating partner connect section](#updating-partner-connect).
 
 1. Once the release is finished. Submit a PR to merge to master. Master build will deploy images, make the final bundle for upload to update stable.
 
-1. Submit PR to merge master back into develop. Approve
+1. Submit PR to merge master back into develop. Approve and merge into develop.
 
 ### Hotfix
 
-Same as release, but change git flow commands to `git flow hotfix`
+Same as release, but change git flow commands to `git flow hotfix` and start from master branch.
 
 ## Manual Releases
 
 ### Prerequesites
 
+To do all the manual actions. You will need to have these additional local commands.
+
 1. [operator-sdk](https://sdk.operatorframework.io/docs/install-operator-sdk/)
 1. [operator-courier](https://github.com/operator-framework/operator-courier)
-1. [gitflow](https://github.com/nvie/gitflow)
 
 ### Release or Bugfix
 
-*Warning*: to do these steps you need pull/push access to master/develop branch.
+*Warning*: to do these steps you need pull/push access to master/develop branch. And admin access to the docker registry. Only to be done if github actions are down.
 
 1. Start from 'develop' branch.
```

Comment:
Numbering may work differently with code-blocks, it shows multiple comments starting at "1.", instead of incrementing to 2, 3 ... . I'll simply add a comment to each line where this shows up.